Actions/Deliver/Deliver.ps1

```diff
@@ -395,18 +405,25 @@ try {
         elseif ($deliveryTarget -eq "AppSource") {
             $projectSettings = ReadSettings -baseFolder $baseFolder -project $thisProject
             $projectSettings = AnalyzeRepo -settings $projectSettings -baseFolder $baseFolder -project $thisProject -doNotCheckArtifactSetting -doNotIssueWarnings
+            # Use old settings and issue warnings
+            'continuousDelivery','mainAppFolder','productId' | ForEach-Object {
+                if ($projectSettings.Keys -contains "AppSource$_") {
+                    OutputWarning "Using AppSource$_ in $thisProject/.AL-Go/settings.json is deprecated. Use deliverToAppSource.$_ instead."
+                    $projectSettings.deliverToAppSource."$_" = $projectSettings."AppSource$_"
```

Comment:
Maybe also mention in the message that if, for example, `AppSourceProjectId` is defined both in `deliverToAppSource` and as a separate setting, the former's value will be used (even when the setting is deprecated).